### PR TITLE
DataGrid Remote Reordering demo: add dropFeedbackMode

### PR DESCRIPTION
**Pull request by pavel-2020**
[Trello](https://trello.com/c/nGCwQqu3/1498-grid-extend-the-remote-reordering-demo)
CP: https://github.com/DevExpress/devextreme-demos/pull/680

### CODE REVIEW

File: JSDemos/Demos/DataGrid/RemoteReordering/description.md [hunk 1]
```diff
@@ -1,12 +1,13 @@
 This demo shows how to use drag and drop to reorder records stored on the server. This functionality requires that records' order indexes are in an individual data field (`OrderIndex` in this demo) and sorted against that field.
 
-Row drag and drop is configured in the [rowDragging][5] object. Set [allowReordering][2] to **true** to enable this feature. When a row is dropped, the [onReorder][0] event handler is called. Use it to update the record's `OrderIndex` on the server.
+Row drag and drop is configured in the [rowDragging][5] object. Set [allowReordering][2] to **true** to enable this feature. To specify the highlight mode of the row's drop position, use the [dropFeedbackMode][6] property. When a row is dropped, the [onReorder][0] event handler is called. Use it to update the record's `OrderIndex` on the server.
```
Comment (RomanTsukanov) [inline JSDemos/Demos/DataGrid/RemoteReordering/description.md]:
```suggestion
Row drag and drop is configured in the [rowDragging][5] object. Set [allowReordering][2] to **true** to enable this feature. To specify the highlight mode of the row's drop position, use the [dropFeedbackMode][6] property. In this demo, it is set to *"push"*: rows move apart with animation to highlight the new position of a row.
```

File: JSDemos/Demos/DataGrid/RemoteReordering/description.md [hunk 2]
```diff
@@ -1,12 +1,13 @@
...
 
-In this demo, we use the **onReorder** function's `toIndex` parameter to obtain the position at which a user dropped the row. The position is then used to get the new order index. The store's [update][1] method sends this index to the server where the records are sorted and returned to the client. Server-side implementation is available in the [ASP.NET Core][3] and [ASP.NET MVC 5][4] versions of this demo under the `DataGridRowReorderingController.cs` tab.
+In this demo, we set **dropFeedbackMode** to *"push"* so that rows could move apart with animation to highlight the new position of a row. We also used the **onReorder** function's `toIndex` parameter to obtain the position at which a user dropped the row. The position is then used to get the new order index. The store's [update][1] method sends this index to the server where the records are sorted and returned to the client. Server-side implementation is available in the [ASP.NET Core][3] and [ASP.NET MVC 5][4] versions of this demo under the `DataGridRowReorderingController.cs` tab.
```
Comment (RomanTsukanov) [inline JSDemos/Demos/DataGrid/RemoteReordering/description.md]:
```suggestion
When a row is dropped, the [onReorder][0] event handler is called. Use it to update the record's `OrderIndex` on the server. In this demo, we use the **onReorder** function's `toIndex` parameter to obtain the position at which a user dropped the row. The position is then used to get the new order index. The store's [update][1] method sends this index to the server where the records are sorted and returned to the client. Server-side implementation is available in the [ASP.NET Core][3] and [ASP.NET MVC 5][4] versions of this demo under the `DataGridRowReorderingController.cs` tab.
```

File: JSDemos/Demos/DataGrid/RemoteReordering/description.md [hunk 3]
```diff
@@ -1,12 +1,13 @@
 This demo shows how to use drag and drop to reorder records stored on the server. This functionality requires that records' order indexes are in an individual data field (`OrderIndex` in this demo) and sorted against that field.
 
-Row drag and drop is configured in the [rowDragging][5] object. Set [allowReordering][2] to **true** to enable this feature. When a row is dropped, the [onReorder][0] event handler is called. Use it to update the record's `OrderIndex` on the server.
+Row drag and drop is configured in the [rowDragging][5] object. Set [allowReordering][2] to **true** to enable this feature. To specify the highlight mode of the row's drop position, use the [dropFeedbackMode][6] property. In this demo, it is set to *"push"*: rows move apart with animation to highlight the new position of a row.
```
Comment (albertov05) [inline JSDemos/Demos/DataGrid/RemoteReordering/description.md]: Can you clarify what you mean by "rows move apart with animation"? If possible, I'd like to see this in action. :)

Comment (pavel-2020) [reply JSDemos/Demos/DataGrid/RemoteReordering/description.md]: @albertov05 
"rows move apart with animation" in the following demo when a user drags a row: https://js.devexpress.com/Demos/WidgetsGallery/Demo/DataGrid/RemoteReordering/jQuery/Light/

Comment (albertov05) [reply JSDemos/Demos/DataGrid/RemoteReordering/description.md]: Thank you. Based on my understanding of the demo, rows seem to move either up or down to create a space to position the row. What do you think about the following suggestion?

`In this demo, it is set to *"push"*: rows move up or down with animation to create space for the new position of the row.`

**Review by RomanTsukanov — APPROVED** [review]

**Pull request merged**